### PR TITLE
Release 12.1.0

### PR DESCRIPTION
**Pull request by wyardley**
#### Pull Request (PR) description
Release 12.1.0

Includes
- #921
- #919 
- #917
- #918

### CODE REVIEW

**Review by smortex — APPROVED** [review]
LGTM

**Comment by root-expert** [comment]
Please regenerate REFERENCE :smile:

**Comment by wyardley** [comment]
> Please regenerate REFERENCE 😄

Not totally sure if you are joking, but I ran it, but didn’t see any changes. The only enhancement is updating the systemd lib version, so think that’s expected?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
...
+[Full Changelog](https://github.com/voxpupuli/puppet-rabbitmq/compare/v12.0.2...v12.1.0)
+
+**Implemented enhancements:**
+
+- bump puppet/systemd to \< 5.0.0 [\#919](https://github.com/voxpupuli/puppet-rabbitmq/pull/919) ([jhoblitt](https://github.com/jhoblitt))
+- Enable usage of custom list of plugins when using static config file [\#917](https://github.com/voxpupuli/puppet-rabbitmq/pull/917) ([enothen](https://github.com/enothen))
+
+**Closed issues:**
+
+- Unable to use custom list inside enabled\_plugins file [\#916](https://github.com/voxpupuli/puppet-rabbitmq/issues/916)
```
Comment (bastelfreak) [inline CHANGELOG.md:18]: I marked that as `skip-changelog`, because the related PR is already n the changelog. Please regenerate the changelog.

**Comment by bastelfreak** [comment]
@wyardley during the `release` rake task we now check if the REFERENCE.md is up2date. If you see any changes locally, please include it in this PR.

**Comment by root-expert** [comment]
> > Please regenerate REFERENCE smile
> 
> Not totally sure if you are joking, but I ran it earlier when creating the release, but didn’t see any changes. The only enhancement is updating the systemd lib version, so think that’s expected?

Seems like https://github.com/voxpupuli/puppet-rabbitmq/pull/917 is containing changes to the parameters and I didn't see any commit updating the REFERENCE that's why I pointed it out. But if there are no changes then :thinking: :man_shrugging:

**Comment by wyardley** [comment]
Yeah, I always run `rake strings:reference`, but will try one more time.

**Comment by bastelfreak** [comment]
if you're lazy, you can also run the `reference` rake task. Saves a few keystrokes!

**Comment by wyardley** [comment]
> if you're lazy, you can also run the reference rake task. Saves a few keystrokes!

Oh, I had run `bundle exec rake strings:generate` instead of `bundle exec rake strings:generate:reference`, which looks like it generated the html docs structure.

All updated.

**Review by bastelfreak — APPROVED** [review]

**Comment by wyardley** [comment]
@bastelfreak btw, this has happened before, but somehow when I run the release task off of the main branch from the upstream repo, despite having just generated the reference, I get a bunch of diffs, looking like it expects a different format.

Can you confirm that the version that's checked in currently is the right / preferred markdown flavor?

```diff
+++ b/REFERENCE.md
@@ -1,15 +1,16 @@
 # Reference
+
 <!-- DO NOT EDIT: This document was generated by Puppet Strings -->
 
 ## Table of Contents
 
-**Classes**
+### Classes
 
-_Public Classes_
+#### Public Classes
 
 * [`rabbitmq`](#rabbitmq): A module to manage RabbitMQ
 
-_Private Classes_
+#### Private Classes
```
etc.

**Pull request merged**